webpack/components/extensions/HostDetails/Tabs/__tests__/errataTab.test.js

```diff
@@ -81,11 +81,10 @@ let searchDelayScope;
 let autoSearchScope;
 
 beforeEach(() => {
-  // jest.resetModules();
   const { results } = mockErrataData;
   [firstErrata, , thirdErrata] = results;
-  searchDelayScope = mockSetting(nockInstance, 'autosearch_delay', 500);
-  autoSearchScope = mockSetting(nockInstance, 'autosearch_while_typing', true);
+  searchDelayScope = mockSetting(nockInstance, 'autosearch_delay', 0);
```

Comment:
I guess the idea here is that we're not testing the auto_delay functionality in all of these tests, so why use a value of 500 at all?

Reply:
500 is default yeah, so we avoid using the delay. 
In most cases that delay is never actually seen in the tests, but I did notice a small reduction in test times. 
The main change was the` 'autosearch_while_typing', true);` bit. 
That makes all of the tests use the autcompleteSearch functionality as it is seen today within contenViews.